### ISSUE TITLE
Doesn't give any results

### ISSUE THREAD

**Issue by knissophiliac** [jasontalon/fb-events-lookup#1]
The herokuapp is not giving any results. I couldn't figure it out why this is so. Could you check it out?

**Comment by jasontalon** [comment]
Yes, I noticed that too. Found out that the facebook api is expecting a new parameter which is not present in my current implementation. Will try to figure it out once i have time.

**Comment by gandolfi974** [comment]
Hi, thanks for your work. your project is it dead or standby ? i try to use puppeteer to list facebook event from a city but it's very difficult. Do you have an idea or a project like this  ?

**Comment by jasontalon** [comment]
Hi, @gandolfi974. The app is not working anymore as the API used for fetching events and security measures has been changed since the project is made.
